### DISCUSSION TITLE
重启安装的应用直接报错服务内部signal: segmentation fault，重启服务器也是这样，mysql和giea都是

### DISCUSSION THREAD

**Discussion by huanxiaa** [1Panel-dev/1Panel#4159] [Q&A]
![微信图片_20240311215858](https://github.com/1Panel-dev/1Panel/assets/125745270/c848e82e-a489-4de6-8fd1-3685b8fd9d59)

**Comment by wanghe-fit2cloud** [comment]
进入 `主机` - `终端` 页面，`cd` 命令到 MySQL 安装目录，然后手动执行 `docker-compose restart` 命令，先查看一下结果。
> 已启动后面的 `文件夹` 按钮可查看应用安装目录。
